velox/docs/develop/testing/fuzzer.rst

```diff
@@ -116,12 +116,49 @@ Fuzzers support a number of powerful command line arguments.
 
 * ``–-batch_size``: The size of input vectors to generate. Default is 100.
 
-In addition, Aggregation Fuzzer supports:
+There are also arguments that toggle certain fuzzer features:
+
+* ``--retry_with_try``: Retry failed expressions by wrapping it using a try() statement. Default is false.
+
+* ``--disable_constant_folding``: Disable constant-folding in the common evaluation path. Default is false.
+
+* ``--enable_variadic_signatures``: Enable testing of function signatures with variadic arguments. Default is false.
+
+* ``--velox_fuzzer_enable_complex_types``: Enable testing of function signatures with complex argument or return types. Default is false.
+
+* ``--lazy_vector_generation_ratio``: Specifies the probability with which columns in the input row vector will be selected to be wrapped in lazy encoding (expressed as double from 0 to 1). Default is 0.0.
+
+* ``--velox_fuzzer_enable_column_reuse``: Enable generation of expressions that re-use already generated columns. Default is false.
+
+* ``--velox_fuzzer_enable_expression_reuse``: Enable re-use already generated expression. Currently it only re-uses expressions that do not have nested expressions. Default is false.
```

Comment:
nit: same as above. E.g., allow one subexpression to appear multiple times in a generated expression.